defaults/azure_config.yaml

```diff
@@ -17,7 +17,6 @@ availability_zone: ''
 root_disk_size_monitor: 50  # GB, remove this field if default disk size should be used
 root_disk_size_db: 30  # GB, increase root disk for larger swap (maximum: 16G)
 root_disk_size_loader: 30  # GB, Increase loader disk in order to have extra space for a larger swap
```

Comment:
No change of the size like in AWS config?

Reply:
fixed